CalculateBetweenBiggestElementAndSmallestElement.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace CoherentSolutionsTask1
+{
+    public class CalculateBetweenBiggestElementAndSmallestElement
+    {
+        public void SumOfBiggestElementAndSmallest()
+        {
+            List<int> array = new List<int>();
+            Console.WriteLine("Enter the how much numbers will you input: ");
+            int imputOfNumber = int.Parse(Console.ReadLine());
+            for (int index = 0; index < imputOfNumber; index++)
+            {
+                Console.WriteLine("Input "+(index+1)+" number: ");

```

Comment:
add spaces for readability 
`
 Console.WriteLine("Input "+ (index + 1) + " number: ");`